scripts/responses.coffee

```diff
@@ -43,10 +43,11 @@ class ResponsesManager
   sendMessage: (msg, response) ->
       if !@isMuted msg
           msg.send response
-
+          
 module.exports = (robot) ->
 
   responses = new ResponsesManager
+  @count
```

Comment:
not positive, but i think if you want it to be persistent, you have to add `count` to the responseManager class and initialize it there